astropy/utils/iers/tests/test_iers.py

```diff
@@ -68,6 +69,14 @@ def test_open_filename(self):
         with pytest.raises(FILE_NOT_FOUND_ERROR):
             iers.IERS.open('surely this does not exist')
 
+    @remote_data
+    def test_open_network_url(self):
+        iers.IERS_A.close()
+        iers.IERS_A.open(IERS_A_EXCERPT_NETWORK)
+        assert iers.IERS_A.iers_table is not None
+        assert isinstance(iers.IERS_A.iers_table, QTable)
+        iers.IERS_A.close()
```

Comment:
Really this could just as well use a `file://` URL for the test.  This doesn't need to test that `download_file` works on remote files (it already has its own tests).